client/controllers/incidentReports/incidentForm.coffee

```diff
@@ -82,43 +90,62 @@ Template.incidentForm.onRendered ->
   @$('#add-incident').parsley()
   #Update the validator when Blaze adds incident type related inputs
   @autorun =>
-    @incidentType.get()
     @locations.get()
     Meteor.defer =>
       @$('#add-incident').parsley().reset()
 
+  @autorun =>
+    switch @incidentType.get()
+      when 'activeCount' then @dateRangeType.set("day")
+      when 'cumulativeCaseCount' then @dateRangeType.set("day")
+      when 'cumulativeDeathCount' then @dateRangeType.set("day")
+
 Template.incidentForm.helpers
   incidentData: ->
     Template.instance().incidentData
 
   incidentStatusChecked: (status) ->
     status is Template.instance().incidentStatus.get()
 
-  incidentTypeChecked: (type) ->
-    type is Template.instance().incidentType.get()
-
   articles: ->
     Template.instance().data.articles
 
   showCountForm: ->
     type = Template.instance().incidentType.get()
-    type is 'cases' or type is 'deaths'
+    type in [
+      'caseCount'
+      'deathCount'
+      'cumulativeCaseCount'
+      'cumulativeDeathCount'
+      'activeCount'
+    ]
 
   showOtherForm: ->
-    Template.instance().incidentType.get() is 'other'
+    Template.instance().incidentType.get() is 'specify'
+
+  showRangeTab: ->
+    type = Template.instance().incidentType.get()
+    type in [
+      'caseCount'
+      'deathCount'
+      'specify'
+    ]
```

Comment:
Could we avoid declaring the `type` variable since its only referenced once? 
Also on line 105.